scripts/publish.sh

```diff
@@ -318,13 +318,13 @@ sync_latest_image() {
 
 	make_repo_public ${region}
 
-	ssm_parameters=$(aws ssm get-parameters --names "/aws/service/aws-for-fluent-bit/${AWS_FOR_FLUENT_BIT_VERSION_DOCKERHUB}" --region ${region})
+	ssm_parameters=$(aws ssm get-parameters --names "/aws/service/aws-for-fluent-bit/${AWS_FOR_FLUENT_BIT_VERSION_DOCKERHUB}" --region ${region} || true)
 	invalid_parameter=$(echo $ssm_parameters | jq .InvalidParameters[0])
 	if [ "$invalid_parameter" != 'null' ]; then
 		publish_ssm ${region} ${account_id}.dkr.ecr.${region}.${endpoint}/aws-for-fluent-bit ${AWS_FOR_FLUENT_BIT_VERSION_DOCKERHUB}
 	fi
 
-	ssm_parameters=$(aws ssm get-parameters --names "/aws/service/aws-for-fluent-bit/stable" --region ${region})
+	ssm_parameters=$(aws ssm get-parameters --names "/aws/service/aws-for-fluent-bit/stable" --region ${region} || true)
```

Comment:
Actually I don't see how this is better, this means the script will continue on if we are not allow-listed for SSM rather than just failing fast? 

Isn't that not ideal? Isn't failing fast better? Or am I misunderstanding?

Reply:
The existing code handles missing parameters well via invalid parameters return:

```
147dda285fd4:aws-for-fluent-bit wppttt$ aws ssm get-parameters --names "/tmp/service/aws-for-fluent-bit/stable" --region us-west-2
{
    "Parameters": [],
    "InvalidParameters": [
        "/tmp/service/aws-for-fluent-bit/stable"
    ]
}
147dda285fd4:aws-for-fluent-bit wppttt$ echo $?
0
```

It does not handle invalid public parameters well, but we want that, to tell us if we messed up some setup:

```
147dda285fd4:aws-for-fluent-bit wppttt$ aws ssm get-parameters --names "/aws/service/does-not-exit/stable" --region us-west-2

An error occurred (AccessDeniedException) when calling the GetParameters operation: No access to "/aws/" namespace: aws/service/does-not-exit is not a valid namespace
147dda285fd4:aws-for-fluent-bit wppttt$ echo $?
254
```

So I think the existing code seems better?